src/elm/Page/Onboarding.elm

```diff
@@ -172,6 +183,50 @@ update msg env model =
                                 V.add [ TravelCardField ] (errorToString error) model.validationErrors
                         }
 
+        RegisterConsents ->
+            if not (Set.isEmpty model.consents) then
+                case validateEmail True model of
+                    Ok _ ->
+                        PageUpdater.fromPair
```

Comment:
Skip / Back skaper flere problemer her. Om jeg lagrer consent og går tilbake etterpå for å angre (oppheve consent) så blir det ikke gjort noe og consent er fortsatt gitt selv om brukere tror de ikke har gitt consent. Det kan være problematisk for personvern her.

Reply:
Hmm, ja. Da kommer det store spørsmålet: skal vi slette consent eller sette consent til false?

Reply:
Ja, det er ikke helt rett frem. Det er mulig vi burde utvide grensesnittet på et tidspunkt, men i dette tilfellet blir vel det rett å slette kanskje? For du trekker samtykket du ga for få minutter siden.

Reply:
Jeg sjekket nå at dersom du setter choice=false så blir epost ikke lagret på samtykket. I Sørvis står det at samtykket er avslått. Så hvis du først registrerer med choice=true og en epost og etterpå på nytt med choice=false så forsvinner epost. Dette er vel grei skuring?

Reply:
Ja, bra. Om de fjerner epost så høres det bra ut tenker jeg! Godt nytt

Reply:
Angående consent og tilbake, hvordan ønsker vi å løse følgende scenario:

- Brukeren gir samtykke, kommer på neste side
- Brukeren trykker på tilbake, havner da på samtykke-siden
- Brukeren trykker hopp over

Skal samtykkene fjernes nå?

Reply:
Jeg tenker i alle fall at da blir ikke noe lagret. Så da består det valget du satte sist. Men jeg ser at det ikke er helt rett frem.

Reply:
Enig med Mikael. Neste lagrer - hopp over lagrer ikke